pkg/keystone/route.go

```diff
@@ -0,0 +1,35 @@
+package keystone
+
+import (
+	routev1 "github.com/openshift/api/route/v1"
+	comv1 "github.com/openstack-k8s-operators/keystone-operator/pkg/apis/keystone/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	intstr "k8s.io/apimachinery/pkg/util/intstr"
+)
+
+// Route func
+func Route(cr *comv1.KeystoneAPI, cmName string) *routev1.Route {
+
+	labels := map[string]string{
+		"app": "keystone-api",
+	}
+	serviceRef := routev1.RouteTargetReference{
+		Kind: "Service",
+		Name: "keystone",
+	}
+	routePort := &routev1.RoutePort{
+		TargetPort: intstr.FromInt(5000),
```

Comment:
I had done this slightly differently. Instead of hard coding the port here I referenced the: targetPort: api in the service (which already has port 5000 hard coded). Either way is probably fine I guess as I'd grep for it.